contrib/cray/Jenkinsfile.verbs

```diff
@@ -404,6 +408,24 @@ pipeline {
                 sh "make install"
             }
         }
+        stage("Install OFIWG Libfabric Build") {
+            when {
+                allOf {
+                    expression { currentBuild.currentResult == 'SUCCESS' } ;
+                    expression { isOfiwgBuild() } ;
+                    expression { env.BRANCH_NAME == 'master' } ;
+                }
+            }
+            environment {
+                LIBFABRIC_INSTALL_PATH="${LIBFABRIC_BUILD_PATH + '/' + 'OFIWG_' + GIT_DESCRIPTION}"
```

Comment:
I think it would be ideal if there is only one 'install libfabric build' stage. Can we set the value of the environment variable based on the isOfiwgBuild variable? That should allow us to reuse the same code without duplicating it. Same goes for the deploy section.

Reply:
The original stages, check to see if it is an internal build.  Whereas, in the new stages, it checks to see if it is an OFIWG build.  The second part of this is the original stages check to see if it is building for a Tag.  If we used the same stage for both internal builds and OFIWG builds, we could end up building OFIWG Tag builds.  If that is not a problem, then the new stages could be removed and the INSTALL_PATH could be changed to add OFIWG if it is an OFIWG build.